internal/variable/variable.go

```diff
@@ -281,6 +281,15 @@ func WriteTerraformVars(dir string, vars []*Variable) error {
 			b.WriteString(" = ")
 			if v.HCL {
 				b.WriteString(v.Value)
```

Comment:
Here we are using EOT as the base delimiter for the heredoc, and optionally adding extra Ts to it if the delimiter is already contained by the value.